From b57a49304351486eaec7530efe3c6542a50c9b4f Mon Sep 17 00:00:00 2001
From: Chris Gorman <chrisjohgorman@gmail.com>
Date: Wed, 26 Jun 2024 11:22:57 -0400
Subject: [PATCH] Add .py extension for gr_filter_design so it will launch from
 grc

- needed for windows as gr_filter_design is named gr_filter_design.py
---
 grc/gui/Application.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grc/gui/Application.py b/grc/gui/Application.py
index 759fe5c09..3a753ffbd 100644
--- a/grc/gui/Application.py
+++ b/grc/gui/Application.py
@@ -823,7 +823,7 @@ class Application(Gtk.Application):
             Actions.ELEMENT_CREATE()
 
         elif action == Actions.TOOLS_RUN_FDESIGN:
-            subprocess.Popen('gr_filter_design',
+            subprocess.Popen('gr_filter_design.py',
                              shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
 
         else:
-- 
2.45.2

